nginx.conf

```diff
@@ -38,15 +38,26 @@ http {
   			proxy_pass http://$VulnerableApplication:80;
         }
         
-        location /VulnerabilityDefinitions {
+        location ~ ^/(?:VulnerabilityDefinitions|scanner\/sast)$ {
         	default_type 'application/json';
         	content_by_lua_block {
-        		local vulnerableAppResponse = ngx.location.capture("/VulnerableApp/VulnerabilityDefinitions")
-                local vulnerableAppJspResponse = ngx.location.capture("/VulnerableApp-jsp/VulnerabilityDefinitions")
-        		local vulnerableAppPhpResponse = ngx.location.capture("/VulnerableApp-php/VulnerabilityDefinitions")
+        		local vulnerableAppResponse = ngx.location.capture("/VulnerableApp" .. ngx.var.uri)
+                local vulnerableAppJspResponse = ngx.location.capture("/VulnerableApp-jsp" .. ngx.var.uri)
+        		local vulnerableAppPhpResponse = ngx.location.capture("/VulnerableApp-php" .. ngx.var.uri)
 				ngx.say(require("vulnerableapp_utility").merge_vulnerability_information(vulnerableAppResponse, vulnerableAppJspResponse, vulnerableAppPhpResponse))
     		}
         }
+
+		location /scanner/dast {
+        	default_type 'application/json';
+        	content_by_lua_block {
+        		local vulnerableAppResponse = ngx.location.capture("/VulnerableApp/scanner")
+                local vulnerableAppJspResponse = ngx.location.capture("/VulnerableApp-jsp" .. ngx.var.uri)
```

Comment:
let's add full URL instead of using `.. ngx.var.uri` i.e. "/VulnerableApp-jsp/scanner/dast" and "/VulnerableApp-php/scanner/dast"